docs/widgets.md

```diff
@@ -12,6 +12,28 @@ The widget appearance depends on the selected style. The following styles are av
 
 See [Selecting a Widget Style](#selecting-a-widget-style) for details how to select the style. If no style is selected, `native` is the default. If `native` is not available, `fluent` is the default.
 
+## structs
```

Comment:
could you move that to the end of the file?  I think it's good to have the widget first

(Alternative is to put that documentation in the paragraph of the table view.)